lib/model/game_end.dart

```diff
@@ -0,0 +1,14 @@
+enum GameEnd {
+  NEWGAME,
+  EXIT
+}
+
+extension GameEndExt on GameEnd {
+  String getDescription() {
+    if (this == GameEnd.NEWGAME) {
+      return "New Game";
+    } else {
+      return "Exit";
+    }
+  }
+}
```

Comment:
This isn't being used currently - not sure what we'd need it for? Can probs remove it

Reply:
Yeah I wasn't sure if I wanted to go that route like the scry page, or just hard code in some options like the main menu. Guess it's a stylistical choice